src/13_file_io.py

```diff
@@ -8,11 +8,21 @@
 # Open up the "foo.txt" file (which already exists) for reading
 # Print all the contents of the file, then close the file
 
+f = open('foo.txt')
+read_data = f.read()
+print(read_data)
+f.close()
+
 # YOUR CODE HERE
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make 
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+f = open("bar.txt", "w")
+f.write("This is line one, yo\n")
+f.write("This is line two, yo\nThis is line three, yo\n")
+f.close()
```

Comment:
Can be simplified to this:

fp = open("bar.txt", "w")

fp.write("""Line 1
            Line 2
            Line 3""")

fp.close()

Reply:
This seems useful, though it does pick up any indentation :joy: